app/models/tattoo.rb

```diff
@@ -1,7 +1,7 @@
 class Tattoo < ActiveRecord::Base
   has_many :reviews, dependent: :destroy
   belongs_to :user
-  validates :title, presence: true
+  validates :title, presence: true, length: { maximum: 20 }
```

Comment:
I want to see updates in the `tattoo_spec` to reflect this new length validation!